blockchain/checkpoints.go

```diff
@@ -98,10 +98,7 @@ func (b *BlockChain) maybeUpdateMostRecentCheckpoint(node *blockNode) {
 func isNonstandardTransaction(tx *dcrutil.Tx) bool {
 	// Check all of the output public key scripts for non-standard scripts.
 	for _, txOut := range tx.MsgTx().TxOut {
-		scriptClass := txscript.GetScriptClass(txOut.Version, txOut.PkScript)
-		if scriptClass == txscript.NonStandardTy {
-			return true
-		}
+		return standalone.IsNonStandardScript(txOut.Version, txOut.PkScript)
```

Comment:
This is not in a consensus code path and therefore is perfectly fine to use `txscript.GetScriptClass` and therefore shouldn't be updated.

As an aside, I would also prefer `standalone` not have standardness-related things in there since I would really like to keep consensus things more clearly separated.